src/components/transaction-list/TransactionList.js

```diff
@@ -308,7 +312,15 @@ export default function TransactionList({
 
   const data = useMemo(
     () => ({
-      requests,
+      requests: requests.map(r => {
```

Comment:
If we do this override in the TransactionList, will we need the same in the ActivityList for non-native txn list users? (and perhaps is it better to do this sort of overriding right at the receipt of the peer metadata payload instead of in various places)

Reply:
It's already done in the Activity list too